src/ftof.cpp

```diff
@@ -0,0 +1,59 @@
+/*! \file ftof.cpp
+ *
+ * \brief Contains functions to transfer EigenMatrix and EigenVector to FASP
+ *  or to tranfer FASP to Fenics
+ */
+
+#include "ftof.h"
+
+using namespace dolfin;
+
+// Link the dolfin::EigenMatrix Mat_A to dCSRmat format from FASP
+void EigenMatrixTOdCSRmat( dCSRmat* dCSR_A, const EigenMatrix* mat_A)
```

Comment:
std::get works for me!

So are you ok merging ?

On Sun, Oct 4, 2015 at 4:36 PM, Maximilian Metti <notifications@github.com>
wrote:

> In src/ftof.cpp
> <https://github.com/thepnpsolver/modular-pnp/pull/3#discussion_r41102076>:
>
> > @@ -0,0 +1,59 @@
> > +/*! \file ftof.cpp
> > + *
> > + * \brief Contains functions to transfer EigenMatrix and EigenVector to FASP
> > + *  or to tranfer FASP to Fenics
> > + */
> > +
> > +#include "ftof.h"
> > +
> > +using namespace dolfin;
> > +
> > +// Link the dolfin::EigenMatrix Mat_A to dCSRmat format from FASP
> > +void EigenMatrixTOdCSRmat( dCSRmat* dCSR_A, const EigenMatrix* mat_A)
>
> Yeah, it should be ``std::get`` actually.
> … <#1503491a9ca6c89a_>
> On Oct 4, 2015, at 08:17, Arthur Bousquet <notifications@github.com>
> wrote: In src/ftof.cpp: > @@ -0,0 +1,59 @@ > +/*! \file ftof.cpp > + * > +
> * \brief Contains functions to transfer EigenMatrix and EigenVector to FASP
> > + * or to tranfer FASP to Fenics > + */ > + > +#include "ftof.h" > + >
> +using namespace dolfin; > + > +// Link the dolfin::EigenMatrix Mat_A to
> dCSRmat format from FASP > +void EigenMatrixTOdCSRmat( dCSRmat* dCSR_A,
> const EigenMatrix* mat_A) #1
> <https://github.com/thepnpsolver/modular-pnp/pull/1> : I am not using the
> boost library. boost::tuple does not work for me. #2
> <https://github.com/thepnpsolver/modular-pnp/pull/2>: I am agree with you
> #3 <https://github.com/thepnpsolver/modular-pnp/pull/3>: just personal
> preference when it's a pointer I like to put it in arguments but I really
> don't mind to change it as a function. Up to you. … — Reply to this email
> directly or view it on GitHub.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/thepnpsolver/modular-pnp/pull/3/files#r41102076>.
>